bench/app.py

```diff
@@ -171,42 +170,80 @@ def get(self):
 	def remove(self):
 		active_app_path = os.path.join("apps", self.repo)
 		archived_path = os.path.join("archived", "apps")
-		archived_name = get_available_folder_name(
-			f"{self.repo}-{date.today()}", archived_path
-		)
+		archived_name = get_available_folder_name(f"{self.repo}-{date.today()}", archived_path)
 		archived_app_path = os.path.join(archived_path, archived_name)
 		log(f"App moved from {active_app_path} to {archived_app_path}")
 		shutil.move(active_app_path, archived_app_path)
 
 	@step(title="Installing App {repo}", success="App {repo} Installed")
-	def install(self, skip_assets=False, verbose=False, restart_bench=True):
+	def install(self, skip_assets=False, verbose=False, resolved=False, restart_bench=True):
 		import bench.cli
 		from bench.utils.app import get_app_name
 
 		verbose = bench.cli.verbose or verbose
 		app_name = get_app_name(self.bench.name, self.repo)
-
-		# TODO: this should go inside install_app only tho - issue: default/resolved branch
-		setup_app_dependencies(
-			repo_name=self.repo,
-			bench_path=self.bench.name,
-			branch=self.tag,
-			verbose=verbose,
-			skip_assets=skip_assets,
-		)
+		if not resolved and self.repo != "frappe":
+			click.secho(
+				f"Ignoring dependencies of {self.name} to install dependencies use --resolve-deps",
```

Comment:
Handle this bit with the setup requirements command too. Perhaps add a `--resolve-deps` option too and don't show this message when called from there anyway

Reply:
Not showing the message when called from setup requirements.